### ISSUE TITLE
The tool fails for wasmer and doesn't give much information

### ISSUE THREAD

**Issue by williamh** [cardoe/cargo-ebuild#29]
reproduce as follows:

```
whubbs@whubbs1 ~ $ git clone git@github.com:wasmerio/wasmer
Cloning into 'wasmer'...
remote: Enumerating objects: 22, done.
remote: Counting objects: 100% (22/22), done.
remote: Compressing objects: 100% (22/22), done.
remote: Total 70416 (delta 5), reused 6 (delta 0), pack-reused 70394
Receiving objects: 100% (70416/70416), 57.83 MiB | 7.92 MiB/s, done.
Resolving deltas: 100% (56455/56455), done.
whubbs@whubbs1 ~ $ cd wasmer
/home/whubbs/wasmer
whubbs@whubbs1 wasmer $ git checkout 0.11.0
...
Note: switching to '0.11.0'.

...

HEAD is now at 8f50dab0f Trying to improve release process
whubbs@whubbs1 wasmer $ cargo-ebuild ebuild
failed to parse manifest at `/home/whubbs/wasmer/Cargo.toml`
whubbs@whubbs1 wasmer $
```

Any help would be appreciated.

Thanks,

William

**Comment by cardoe** [comment]
That's actually an internal error coming from Cargo itself. This crate uses an older version of Cargo and it's likely related to that. I'm in the process of switching a lot of usage over to cargo_toml and cargo_metadata. I've been burned before by cargo_metadata removing data necessary but I'll attempt to trust it again.

**Comment by cardoe** [comment]
So I've created a branch https://github.com/cardoe/cargo-ebuild/tree/change-up-the-parse which might do a better job. Any chance you can give it a whirl? I'm on slow, spotty internet until after the new year so I cannot test until then.

**Comment by RuTom** [comment]
So I gave the branch a try because I faced a similar problem and it does produce an ebuild.
Haven't tested if the wasmer build would compile but it did create a valid ebuild.

**Comment by cardoe** [comment]
I've gone ahead and released cargo-ebuild 0.3.0 with these changes.
